app/pods/newsletters/template.hbs

```diff
@@ -8,15 +6,15 @@
       <div class="auk-u-m-8">
         <DataTable
           data-test-route-newsletters-data-table={{true}}
-          @content={{this.model}}
+          @content={{@model}}
```

Comment:
Did you get a warning for this one? I think we use `this.model` in most places atm. This kind of change occurs a couple of times in this PR.

Reply:
Not a warning, it's just [stated in the ember docs](https://guides.emberjs.com/release/routing/specifying-a-routes-model/) to use `@model` in the template and `this.model` in the controller, which is why I tend to change it, though both do seem to work.

Although `@model` does seem to be used in some pods, most of the app uses `this.model`, so I can revert those changes for consistency.

Reply:
No, no problem if `@model` is preferred by the official docs. Thanks for the pointer :pray: